chainstate/src/detail/ban_score.rs

```diff
@@ -61,18 +60,19 @@ impl BanScore for BlockError {
             BlockError::TxIndexConstructionError(_) => 100,
             BlockError::PoSAccountingError(err) => err.ban_score(),
             BlockError::EpochSealError(err) => err.ban_score(),
-            BlockError::BlockHeightTooBig(_) => 0,
+            BlockError::BlockTooDeepToInvalidate(_) => 0,
+            BlockError::BestChainCandidatesError(_) => 0,
+            BlockError::BlockNotFound(_) => 0,
```

Comment:
Please make the error more verbose by including the context in the name. This runs the danger of someone else using this error without noticing it has 0 ban score.